### ISSUE TITLE
[BUG] PodDisruptionBudget - type bug

### ISSUE THREAD

**Issue by zekth** [stackrox/kube-linter#562]
**System info:**
 - OS: MacOs
 - kube-linter: 0.6.3

**Describe the bug**

As stated here: https://kubernetes.io/docs/tasks/run-application/configure-pdb/#rounding-logic-when-specifying-percentages

> Values for minAvailable or maxUnavailable can be expressed as integers or as a percentage.

However when using percentage an error is triggered

**To Reproduce**
`kube-linter lint `

**Sample YAML input**

```yaml
apiVersion: policy/v1
kind: PodDisruptionBudget
metadata:
  name: foo
spec:
  maxUnavailable: "20%"
```
**Expected behavior**
`pdb-max-unavailable` should not trigger in case of percentage

Note: This seems to work on main branch

**Comment by janisz** [comment]
> Note: This seems to work on main branch

Fixed:
- https://github.com/stackrox/kube-linter/pull/546
